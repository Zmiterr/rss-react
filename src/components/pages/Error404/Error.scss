.error {
	&__link {
		font-size: 18px;
	}
}
