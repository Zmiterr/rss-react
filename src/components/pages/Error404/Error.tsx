import React, { ReactElement } from 'react';
import { Link } from 'react-router-dom';

import './Error.scss';

const Error = (): ReactElement => {
	return (
		<div className="error">
			<h2>404. Not found</h2>
			<Link to="/" className="error__link">
				Click! To redirect on Home page
			</Link>
		</div>
	);
};

export default Error;
