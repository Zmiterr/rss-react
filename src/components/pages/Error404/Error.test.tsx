import React from "react";
import Error from './Error';
//test deps
import Enzyme, { shallow, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter } from 'react-router-dom';


Enzyme.configure({ adapter: new Adapter() })


describe("Error component", () => {
	it("should render Error component", () => {
		const component = shallow(<BrowserRouter><Error /></BrowserRouter>);
		expect(component).toMatchSnapshot();
	});
});

