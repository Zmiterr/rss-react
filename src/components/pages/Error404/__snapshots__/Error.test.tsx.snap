// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render Error component 1`] = `ShallowWrapper {}`;
